lib/AST/Type.cpp

```diff
@@ -381,6 +379,10 @@ ExistentialLayout::ExistentialLayout(CanParameterizedProtocolType type)
   containsParameterized = true;
 }
 
+bool ExistentialLayout::isObjCProtocol(ProtocolDecl *P) {
+  return P->isObjC() || P->isSpecificProtocol(KnownProtocolKind::Sendable);
```

Comment:
Should this allow any marker protocol?

Reply:
I don't think so because Sendable is special due to the concurrency, all other marker protocols are not just representable.